pimd/pim6_cmd.c

```diff
@@ -999,6 +999,22 @@ DEFPY (interface_no_ipv6_mroute,
 					    source_str);
 }
 
+DEFPY (interface_ipv6_pim_use_source,
+      interface_ipv6_pim_use_source_cmd,
+       "[no] ipv6 pim use-source X:X::X:X",
```

Comment:
If you make the command:
`[no] ipv6 pim use-source X:X::X:X$source`

you can then use use `source_str` instead of `argv[3]->arg`, which is the more common way if we dealing with CLI args nowadays.